docs/hugo.toml

```diff
@@ -113,6 +113,9 @@ github_project_repo = "https://github.com/cilium/tetragon"
 # Specify a value here if your content directory is not in your repo's root directory
 github_subdir = "docs"
 
+# URL of the Star Wars demo app.
+demo_app_url = "https://raw.githubusercontent.com/cilium/cilium/v1.15.0-pre.1/examples/minikube/http-sw-app.yaml"
+
```

Comment:
Thanks, I would like to merge that but why did you move this piece of TOML in the file with no change? Did you forget something here?

If you want to update that, can you let it at the lines where it was sitting?

Reply:
> why did you move this piece of TOML in the file with no change
I explained this in https://github.com/cilium/tetragon/pull/1867#discussion_r1449573367, PTAL and let me know if anything does not make sense.

Reply:
Wow indeed those 2 params don't work and it's broken, I think it's my fault I didn't realize it! thanks, will fix the other one, let's merge this.